### PR TITLE
Submitting codekeys PCM

### PR DESCRIPTION
**Pull request by EatThePath**
of course should really only be merged after #43 and #44

### CODE REVIEW

File: pcm_codekeys/readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Reloading the configs every keypress could, if there's lots of code, potentially cause performance issues. Removing the module or clearing your config files before releasing a mod is recommended.
+
+# Installing
+
+ * Ensure Plasma Core 0.11.0 is present in your mod.
+ * If you intend to use fennel actions, include the add-fennel for fennel version 1.3.0.
+ * Place either codekeys.fnl or codekeys.lua into your mod's data/scripts/plasma_modules
+ * put any configs in data/config.
+
+Both codekeys.fnl and codekeys.lua are provided. .fnl is how the module was developed, but using the .lua may provide easier debugging if issues arise, even if you have fennel available. There are some weird constructions in the lua due to it's nature as compiler output, these have largely been left in place to ensure equivilent logic.
```
Comment (Goober5000) [inline pcm_codekeys/readme.md]: equivalent

File: pcm_codekeys/readme.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Codekeys module for Plasma Core 0.11.0
+This module enables the attachment of code to hotkeys to facilitate interactive development. 
+
+Actions are defined in `.cfg` files, supporting both pure lua and fennel syntax. prefix lua files with `ckl-` and fennel with 'ckf-' and place them in data/config. Examples of each are provided. So long as a properly named config is indexed by the engine it will be reloaded on every keypress.
```
Comment (Goober5000) [inline pcm_codekeys/readme.md:4]: Every single keypress?  What about a special keypress, like F12, that means 'reload'?

Comment (EatThePath) [reply pcm_codekeys/readme.md:4]: If the objective was just a reload key, then sure that'd be the way to go. The advantage of this approach is that it's easy to tweak to work around any given keybindings, and lets you bind multiple keys if desired. I've had up to four keys with different actions at some points. I don't think there's really any way to accomplish that responsively at runtime without the current structure.

Comment (Goober5000) [reply pcm_codekeys/readme.md:4]: Ok.

**Review by Goober5000 — APPROVED** [review]

**Pull request merged**